src/Services/OrderService/OrderService.Application/obj/Debug/net6.0/OrderService.Application.AssemblyInfo.cs

```diff
@@ -0,0 +1,23 @@
+//------------------------------------------------------------------------------

```

Comment:
Seems that some folders bypassed the .gitignore... You can safely delete bin/obj/debug and some other folders that are generated by your IDE from the repo.